apps/src/storage/dataBrowser/DataOverview.jsx

```diff
@@ -61,6 +61,9 @@ const DataOverview = React.createClass({
       <div id="dataOverview" style={{display: visible ? 'block' : 'none'}}>
         <h4>Data</h4>
 
+        <h5>{msg.dataTabExplanation()} <a id="dataBlocksLink" href="https://code.org/applab/docs/tabledatastorage"><strong>{msg.dataBlocks()}</strong></a> {msg.withinApp()}</h5>
+        <br/>
```

Comment:
unfortunately, because different languages have different grammar, this isn't an i18n (internationalization) friendly way to construct a string. Fortunately, our translation system has the ability to include placeholders in a string. Here is a simple example of the `completedWithoutRecommendedBlock` ui string using a placeholder:
* https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json#L37
* https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/feedback.js#L799-L801

Ideally you would do something like the following:
1. define a `dataBlocksLink` variable to save the `<a id="dataBlocksLink" href="https://code.org/applab/docs/tabledatastorage"><strong>{msg.dataBlocks()}</strong>` snippet from your above code that renders msg.dataBlocks into a string
2. create a single, longer ui string containing your `dataTabExplanation`, then a placeholder for the dataBlocksLink, then the contents of your `withinApp` string
3. inject your `dataBlocksLink` in as the placeholder